renovate.json

```diff
@@ -4,15 +4,17 @@
   ],
```

Comment:
We need a base config that only allows openstack-k8s-operators updates otherwise renovate will try to bump k8s.io as well but we don't want that on stable branches.

 "extends": [
    "github>openstack-k8s-operators/renovate-config:stable.json"
  ],